share/site/duckduckhack/inc/head.tx

```diff
@@ -1,3 +1,5 @@
+<title><: lp("title","DuckDuckHack - an open source community for DuckDuckGo Instant Answers") :></title>
```

Comment:
I like the long one. Doesn't really matter if it overflows the tab, main purpose for being descriptive is in search results.